source/agora/test/Base.d

```diff
@@ -1558,6 +1558,12 @@ private mixin template TestNodeMixin ()
         return new ManagedDatabase(":memory:");
     }
 
+    /// Return a transaction pool backed by an in-memory SQLite db
+    public override TransactionPool makeTransactionPool ()
+    {
+        return new TransactionPool(new ManagedDatabase(":memory:"), &getDoubleSpentSelector);
```

Comment:
Why ? The default version uses `cacheDB`. I actually planned to remove those, as nowadays cacheDB / stateDB is the only place where dependency injection is done.